src/tests/verify_resources.py

```diff
@@ -69,7 +72,12 @@ def get_contacts():
     txt = urlopen(_contacts_url).read()
     xmltree = et.fromstring(txt)
     users = xmltree.findall('User')
-    return dict(map(user_id_name, users))
+    d = dict(map(user_id_name, users))
+    for u in users:
+        cilogonid = u.find('CILogonID')
+        if cilogonid:
+            d[cilogonid] = u.find('FullName').text
```

Comment:
what does this stuff do?

Reply:
the `contacts` dict returned by this function was previously a map of ContactID -> FullName.  This block adds CILogonID -> FullName for the contacts that have a CILogonID, so that both ContactID and CILogonID (if any) will be keys in this contacts dict that map to the FullName.

Given that that was unclear enough for you to ask, i can throw this into a separate, named function.

Reply:
@matyasselmeci, does this help make it clearer?

Reply:
Yeah, it helps.